<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskBarMenu</class>
 <widget class="QWidget" name="TaskBarMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskBarMenu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>TaskBarSearch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="settings_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exit_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="engines_label">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Search Engine:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="engines_list">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="search_bar"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
